### ISSUE TITLE
Local Terraform module cannot be resolved on Windows

### ISSUE THREAD

**Issue by hugorut** [infracost/infracost#1780]
Infracost fails to fetch some local module source paths when running on Windows. The error log gives the following message: 

```go
Failed to load module err: missing module with source 'registry.terraform.io/../../ec2-instance ../../ec2-instance' -  try to 'terraform init' first
```

This indicates that Infracost tries to load it as a registry module. Note this behaviour is resolved if using Linux. 

---
Note for community
Please 👍 if you'd like to upvote this or receive updates.
Add a comment if you'd like to [contribute](https://github.com/infracost/infracost/blob/master/CONTRIBUTING.md).

**Comment by alikhajeh1** [comment]
@exoemay can you possibly upgrade to the [latest CLI version](https://www.infracost.io/docs/#1-install-infracost) released today (v0.10.17)? We've made a lot of improvements/bug fixes since June so I'm wondering if the issue still exists?

**Comment by vdmgolub** [comment]
Closing in favor of #2639, looks like it was working okay before v0.10.23 release.
